src/scripts/main.js

```diff
@@ -1 +1,48 @@
 'use strict';
+
+const taskBlock = document.createElement('div');
+
+taskBlock.setAttribute('data-qa', 'notification');
+document.body.appendChild(taskBlock);
+
+const firstPromise = new Promise(function(resolve, reject) {
+  taskBlock.addEventListener('click', () => {
```

Comment:
```suggestion
  document.addEventListener('click', () => {
```
<img width="727" alt="Screenshot 2023-12-30 at 17 52 41" src="https://github.com/mate-academy/js_promises_practice_DOM/assets/135232001/863f7610-bb48-4273-afdb-4574fb298c54">